cfme/tests/v2v/test_v2v_schedule_migrations.py

```diff
@@ -1,83 +1,78 @@
+        assignee: ytale
+        initialEstimate: 1/2h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
         casecomponent: V2V
-        initialEstimate: 1/4h
-        subcomponent: RHV
-        upstream: yes
+        testSteps:
+            1. Add source and target provider
+            2. Create infra map and migration plan
+            3. Schedule migration plan
     """
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(
-        form_data_vm_obj_single_datastore.form_data)
-
-    @request.addfinalizer
-    def _cleanup():
-        infrastructure_mapping_collection.delete(mapping)
-
-    migration_plan_collection = appliance.collections.v2v_plans
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    src_vm_obj = mapping_data_vm_obj_mini.vm_list[0]
     migration_plan = migration_plan_collection.create(
-        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
-        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
-        vm_list=form_data_vm_obj_single_datastore.vm_list, start_migration=False)
-    view = navigate_to(migration_plan_collection, 'All')
-
-    # Test scheduled Migration
-    view.migration_plans_not_started_list.schedule_migration(migration_plan.name, after_mins=3)
-    soft_assert('Migration scheduled' in view.migration_plans_not_started_list.
-        get_clock(migration_plan.name))
-    view.migration_plans_not_started_list.schedule_migration(migration_plan.name)
-    soft_assert(view.migration_plans_not_started_list.get_clock(migration_plan.name) == '')
-    view.migration_plans_not_started_list.schedule_migration(migration_plan.name, after_mins=3)
-    view.switch_to('In Progress Plans')
-    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
-        message="migration plan is starting, be patient please", delay=30, num_sec=600,
-        handle_exception=True, fail_cond=False)
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        target_provider=provider,
+        vm_list=mapping_data_vm_obj_mini.vm_list,
+        start_migration=False
+    )
+    view = navigate_to(migration_plan_collection, "NotStarted")
+    view.plans_not_started_list.schedule_migration(migration_plan.name)
+    soft_assert("Migration scheduled" in view.plans_not_started_list.get_clock(migration_plan.name))
 
-    # wait until plan is in progress
-    wait_for(func=view.plan_in_progress, func_args=[migration_plan.name],
-        message="migration plan is in progress, be patient please",
-        delay=5, num_sec=1800)
+    # Extra wait_for until schedule time
+    view = navigate_to(migration_plan, "InProgress")
```

Comment:
Required : 
 assert migration_plan.plan_started 
instead of  view = navigate_to(migration_plan, "InProgress")

Reply:
Good idea, tested locally, it works!